skills/lark-whiteboard-cli/SKILL.md

```
@@ -0,0 +1,236 @@
+description: >
+  当用户要求或使用飞书画板绘制架构图、流程图、思维导图、时序图或其他可视化图表时使用此 skill，作为使用 whiteboard-cli 设计图表布局的指南
+compatibility: Requires Node.js 18+
+metadata:
+  requires:
+    bins: ["lark-cli"]
+---
+
+> [!NOTE]
+> **环境依赖**：绘制画板需要 `@larksuite/whiteboard-cli`（画板 Node.js CLI 工具），以及 `lark-cli`（LarkSuite CLI 工具）。
+> 如果执行失败，手动安装后重试：`npm install -g @larksuite/whiteboard-cli@^0.2.0`
+
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)